index.html

```diff
@@ -0,0 +1,278 @@
+
+        <button type="submit" class=" bb sec3rbutton sec3rlink4 tl"><span class="d">See Projects</span></button>
+
+
+ </section>
+   
+        
+  
+
+
+
+
+
+
+ <section class="section6">
+ 
+           <img src="images/b.png" alt="as" class="imagesec4">
+  
+              <h2 class="tonic tm">Multi-Post
+             Stories</h2>
+                     
+            <ul>
+                 <li class="sec3rlink2">
+        
+                    <a href="/someurl" class="sec3rlink31 ils">CANOPY</a>
+                     <b class="sec3p">.</b>
+
+                     <a href="/someurl" class="sec3rlink31">Backen Dev</a>
+                      <b class="sec3p">.</b>
+
+                    <a href="/someurl" class="sec3rlink31">2015</a>
+                  </li>
+  
+               </ul>
+
+     <p class="sec3p1 l1"> A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+              <ul class="tm">
+
+               <li class="sec3rlink23">
+      
+                    <a href="/someurl" class="sec3rlink31 slink">HTML</a>
+      
+
+                    <a href="/someurl" class="sec3rlink31 slink">CSS</a>
+
+      
+                    <a href="/someurl" class="sec3rlink31  slink">JAVASCRIPT</a>
+                </li>
+              </ul>
+
+     <button type="submit" class=" bb sec3rbutton sec3rlink4 tm"><span class="d">See Projects</span></button>
+   </section>
+
+
+   
+
+
+
+        
+  <section class="section7">
+          
+               <h2 class="h2 imelex">About <br>Myself</h2>
+                 <h4  class="sec3p1">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</h4>
+                    
+                <h5 class="ets" >ETS CONNECT</h5>
+        <ul class="social-menue">
+          <li><img src="images/twitter.PNG" alt="abc"></li>
+          <li><img src="images/linkedinicon.PNG" alt="abc"></li>
+          <li> <img src="images/medium.PNG" alt="abc"></li>
+          <li> <img src="images/github.PNG" alt="abc"></li>
+          <li><img src="images/angellist.PNG" alt="abc"></li>
+        </ul>
+             
+      <button type="submit" class=" bb sec3rbutton sec3rlink4 tm bd"><span class="d">GET my resume</span></button>
+
+
+            
+
+    
+
+               
+               <ul class="limage lsdf">
+                    <li><img src="images/dm.PNG" alt="abc" class="lsdks"></li>
+                    <li><img src="images/k.PNG" alt="abc" class="lsdks"></li>
+  
+                    <li><img src="images/l.PNG" alt="abc" class="lsdks"></li>
+  
+                    <li><img src="images/p.PNG" alt="abc" class="lsdks"></li>
+  
+                    <li><img src="images/q.PNG" alt="abc" class="lsdks"></li>
+  
+                    <li><img src="images/s.PNG" alt="abc" class="lsdks"></li>
+               </ul>
+    </section>
+    
```

Comment:
- [ ] As mentioned by the previous reviewer, kindly make sure your project design sticks as much as possible to your chosen template. You are supposed to use the correct margins/paddings, color, font size, button size && interactions, **fixing the broken images,...**

| Figma Design | Your DEsign |
| ----------------- | ------------------- |
| ![about_section_1](https://user-images.githubusercontent.com/83042742/185762455-f4f26dd3-eb9f-4f12-bfc8-82952205eaaf.png) | ![Screenshot from 2022-08-20 20-49-47](https://user-images.githubusercontent.com/83042742/185762461-1182aca6-de84-4867-ba31-76fcd281b230.png) |